### PR TITLE
Rearrange buttons on update dialog

### PR DESCRIPTION
**Pull request by jaelim-ms**
Failed to have scrolling behavior on update dialog, this change is only for rearrange buttons.

### CODE REVIEW

File: MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m [hunk 1]
```diff
@@ -411,26 +411,26 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
         [MSAlertController alertControllerWithTitle:MSDistributeLocalizedString(@"Update available")
                                             message:releaseNotes];
 
+    // Add a "Postpone"-Button
+    [alertController addDefaultActionWithTitle:MSDistributeLocalizedString(@"Postpone")
+                                       handler:^(UIAlertAction *action) {
+                                         MSLogDebug([MSDistribute logTag], @"Postpone the release for now.");
```
Comment (TroubleMakerBen) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: Shouldn't it be `Postpone the UPDATE for now`?

File: MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m [hunk 2]
```diff
@@ -411,26 +411,26 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
...
                                          [MS_USER_DEFAULTS setObject:details.id forKey:kMSIgnoredReleaseIdKey];
                                        }];
 
-    // Add a "Postpone"-Button
-    [alertController addCancelActionWithTitle:MSDistributeLocalizedString(@"Postpone")
+    // Add a "Download"-Button
+    [alertController addCancelActionWithTitle:MSDistributeLocalizedString(@"Download")
                                       handler:^(UIAlertAction *action) {
-                                        MSLogDebug([MSDistribute logTag], @"Postpone the release for now.");
+                                        MSLogDebug([MSDistribute logTag], @"Start download and install the release.");
```
Comment (TroubleMakerBen) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: `Release` or `Update`?

File: MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m (hunk 1 above)
Comment (jaelim-ms) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: Oh.. :cry:

File: MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m (hunk 2 above)
Comment (jaelim-ms) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: Will use `update`

**Pull request merged**